libptkl.c

```diff
@@ -124,7 +124,7 @@ is_pts_fd(const int fd) {
 int
 open(const char * name, int flags, ...) {
 	int ret = libc.open(name, flags);
-	if (strstr(name, "/ptmx")) {
+	if (ret != -1 && strstr(name, "/ptmx")) {
```

Comment:
I agree with preventing invalid file descriptor being add to the descriptor array, however we should not prevent logging. The same apply for all changed chunks.